### ISSUE TITLE
List creation requirements

### ISSUE THREAD

**Issue by ZakGriffith** [scaffold-eth/OP-RetroPGF3-Discovery-Voting#161]
When sharing a list, currently only the name is required.  We should clean up the front end behavior of this modal and also make the description required.

- Make description required
- Make the front end check to see if they have the required fields filled out and give them an error when the try to share it if not.  This way they won't just get a blank default error.

**Comment by electrone901** [comment]
I can help if need it

**Comment by electrone901** [comment]
Are referring to this modal? 

![Screenshot 2023-11-08 at 8 42 05 PM](https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/assets/10853211/b5b503a1-6a9e-431a-b12c-439587e34901)

**Comment by electrone901** [comment]
Do we need to make the list description, impact description, and the tags required?  
If so do we have any number of characters required for each field? I was thinking of something like the pic below.  

![Screenshot 2023-11-08 at 8 54 58 PM](https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/assets/10853211/02649b87-3da3-4330-a2c0-e57b73bd6474)

**Comment by ZakGriffith** [comment]
Yes, we have limits on the list items.  I believe it is 75 chars for the title, and 1000 for the description and impact as seen in this PR 
https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/pull/122

I think we should require the name and description and leave the impact and tags optional. (Using the front end)

**Comment by electrone901** [comment]
Great, working on this
